src/components/App.js

```diff
@@ -0,0 +1,9 @@
+import Calculator from './calculator';
+
+function App() {
+  return (
+    <Calculator />
+  );
+}
+
+export default App;
```

Comment:
- [ ] The App component file should be out of the components root folder, Kindly move it back to the src footer where it was before. It is always the best practice to keep the components fooder for your application component and the app on the root folder. After fix this, you will need to import the Calculator component like this:
```js
import Calculator from ./components/Calculator; // Import Calculator component
```

- [OPTIONAL] It is alway the best pratice to capitalize your compoenent name: eg :`Calculator .js`